convert/terraform/variables.tf

```diff
@@ -0,0 +1,105 @@
+variable "deploy_id" {
+  type        = string
+  description = "Domino Deployment ID."
+}
+
+
+variable "region" {
+  type        = string
+  description = "AWS region for the deployment"
+}
+
+
+variable "grandfathered_creation_role" {
+  type        = string
+  description = "Role CDK used to create EKS"
+}
+
+
+variable "tags" {
+  type        = map(string)
+  description = "Deployment tags."
+}
+
+variable "vpc_id" {
+  type        = string
+  description = "Pre-existing VPC ID for deployment"
+}
+
+variable "public_subnet_ids" {
+  type        = list(string)
+  description = "Pre-existing public subnets ids used with deployment"
+}
+
+variable "private_subnet_ids" {
+  type        = list(string)
+  description = "Pre-existing private subnets ids used with deployment"
+}
+
+variable "pod_subnet_ids" {
+  type        = list(string)
+  description = "Pre-existing private subnets ids used with deployment"
+}
+
+variable "k8s_version" {
+  type        = string
+  description = "EKS cluster k8s version (should match existing)"
+}
+
+variable "ssh_key_path" {
+  type        = string
+  description = "Path to private SSH key"
+}
+
+variable "number_of_azs" {
+  type        = number
+  description = "Number of AZs used in deployment (usually 3)"
```

Comment:
I wonder if it's a bit more descriptive as # of subnets given the convo today? Though I can come back through here and clean it up once the other changes land

Reply:
Makes sense

Reply:
Actually, I'm going to leave it as-is for now since it still maps directly onto the module's variable which the same variable name and similar description wording.